README.md

```diff
@@ -310,7 +309,7 @@ Dampening is applied to future forecasts whenever a forecast is fetched, so fore
 >
 > Retained dampened historical forecasts is a recent change, and may require automation modification to read undampened forecast history instead. See [Reading forecast values in an automation](#reading-forecast-values-in-an-automation) and [Changes](#changes) below.
 
-Per-site and per-half hour dampening is possible only by using service calls or modifying a dampening configration file. See [Granular dampening](#granular-dampening) below.
+Per-site and per-half hour dampening is possible only by using the integration action or modifying a dampening configration file. See [Granular dampening](#granular-dampening) below.
```

Comment:
drop integration

Reply:
I was trying to give more context as to the 'action'.  I've put the correct action name in now which makes it clearer